### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by bool-**

### CODE REVIEW

File: src/store/modules/plex/helpers/PlexClient.js [hunk 1]
```diff
@@ -122,7 +122,8 @@ module.exports = function PlexClient() {
         if (((new Date().getTime() - this.lastSubscribe) > 29000) && !dontSub) {
           // We need to subscribe first!
           try {
-            await this.subscribe(connection);
+            // This causes certain clients to crash and is unused(Fire Devices)
```
Comment (samcm) [inline src/store/modules/plex/helpers/PlexClient.js]: I remember there was a client that stupidly depended on this - maybe Plex Media Player or iOS? Either way, happy to test it out 👌

**Comment by samcm** [comment]
Heya. Looks great - thanks for helping out!

I'll have a test on the weekend and merge/deploy. Thanks again!

File: src/store/modules/plex/helpers/PlexClient.js (hunk 1 above)
Comment (bool-) [reply src/store/modules/plex/helpers/PlexClient.js]: I didn't come across an issue with any of the clients I tested, Roku, iOS, and Fire TV. If there is an issue I suppose I can make a quick change to test for the problematic client and only send it for that one.

Comment (samcm) [reply src/store/modules/plex/helpers/PlexClient.js]: I just tried it with PMP and it no longer connects 😢. Maybe we have a map of client types that send the subscribe command to?

Comment (samcm) [reply src/store/modules/plex/helpers/PlexClient.js]: Actually, I just downloaded the latest PMP and its connecting. I guess Plex is moving away from the subscribe pattern? Who knows. Merging 👍

**Pull request merged**